diff --git a/src/vmm/src/resources.rs b/src/vmm/src/resources.rs
index d29f76740..51f0e4296 100644
--- a/src/vmm/src/resources.rs
+++ b/src/vmm/src/resources.rs
@@ -1530,7 +1530,7 @@ mod tests {
     #[test]
     fn test_set_boot_source() {
         let tmp_file = TempFile::new().unwrap();
-        let cmdline = "reboot=k panic=1 pci=off nomodule 8250.nr_uarts=0";
+        let cmdline = "reboot=k panic=1 pci=off 8250.nr_uarts=0";
         let expected_boot_cfg = BootSourceConfig {
             kernel_image_path: String::from(tmp_file.as_path().to_str().unwrap()),
             initrd_path: Some(String::from(tmp_file.as_path().to_str().unwrap())),
diff --git a/src/vmm/src/vmm_config/boot_source.rs b/src/vmm/src/vmm_config/boot_source.rs
index dc21523af..c74a56620 100644
--- a/src/vmm/src/vmm_config/boot_source.rs
+++ b/src/vmm/src/vmm_config/boot_source.rs
@@ -9,13 +9,12 @@ use serde::{Deserialize, Serialize};
 /// Default guest kernel command line:
 /// - `reboot=k` shut down the guest on reboot, instead of well... rebooting;
 /// - `panic=1` on panic, reboot after 1 second;
-/// - `nomodule` disable loadable kernel module support;
 /// - `8250.nr_uarts=0` disable 8250 serial interface;
 /// - `i8042.noaux` do not probe the i8042 controller for an attached mouse (save boot time);
 /// - `i8042.nomux` do not probe i8042 for a multiplexing controller (save boot time);
 /// - `i8042.dumbkbd` do not attempt to control kbd state via the i8042 (save boot time).
 /// - `swiotlb=noforce` disable software bounce buffers (SWIOTLB)
-pub const DEFAULT_KERNEL_CMDLINE: &str = "reboot=k panic=1 nomodule 8250.nr_uarts=0 i8042.noaux \
+pub const DEFAULT_KERNEL_CMDLINE: &str = "reboot=k panic=1 8250.nr_uarts=0 i8042.noaux \
                                           i8042.nomux i8042.dumbkbd swiotlb=noforce";
 
 /// Strongly typed data structure used to configure the boot source of the
